content/docs/authorization/configuration/powerflex/_index.md

```diff
@@ -58,4 +58,6 @@ Create the karavi-authorization-config secret using this command:
 
     Enable CSM for Authorization and provide the *proxyHost* address
 
-7. Install the CSI PowerFlex driver
\ No newline at end of file
+7. Install the CSI PowerFlex driver
+
+8. (Optional) Install [dellctl](../../../references/cli) to perform Kubernetes administrator commands for more functionalities (e.g., list volumes). Please refer to the [dellctl documentation page](../../../references/cli) for installation steps and command list.
```

Comment:
Rather than saying "more functionalities", can this say "additional capabilities"?

Reply:
fixed.